proto/usr/triton/bin/config.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/bash
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+#
+# Copyright 2020 Joyent Inc.
+#
+#
+# functions for getting USB headnode config values (intended to be sourced
```

Comment:
Yeah, the standard model is still having a USB inserted that we'll boot from. Even for Linux CNs it's a triton boot image with SmartOS on it. We use `loader` to load `ipxe`, which contacts booter, and is given a SmartOS or Linux PI.

Future work would be to add `piadm` (or something like it) so that we can do Linux CNs in packet.

Reply:
A distinct on-board boot device (disk or pool) can probably simulate what you need, but yeah, that's later.

Reply:
Yeah, I want to keep the same model that we have with SmartOS (including using loader), because I think that works really well.